test/helpers/selenium-helper.js

```diff
@@ -87,19 +95,31 @@ class SeleniumHelper {
         return this.findByXpath(`//body//${scope || '*'}//*[contains(text(), '${text}')]`);
     }
 
-    loadUri (uri) {
+    async loadUri (uri) {
```

Comment:
What is the significance of this change?

Reply:
This just allows me to use `await` in the body... IMO that makes it easier to read, but I could switch back to a `.then(...)` chain if you prefer.

Reply:
Ah right sounds good!